docs/hooks/useScaffoldEthBalance.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 9
+---
+
+# useScaffoldEthBalance
+
+Use this hook to read the Ethereum (ETH) balance of a specified address on the StarkNet blockchain.
+
+```ts
+const { value, formatted, symbol } = useScaffoldEthBalance({
+  address: "0xYourAddressHere",
```

Comment:
You can use a random address here instead of a placeholder like this to make it less confusing and more real-life like.